api/src/main/java/io/strimzi/api/kafka/model/status/KafkaClusterRebalanceStatus.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.status;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+/**
+ * Represents a status of the Kafka Cluster Rebalance resource
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({ "conditions", "observedGeneration", "optimizationResult" })
+@EqualsAndHashCode
+@ToString(callSuper = true)
+public class KafkaClusterRebalanceStatus extends Status {
+
+    private static final long serialVersionUID = 1L;
+
+    private String optimizationResult;
```

Comment:
Why would we embed some JSON within a string in the JSON status? We should be able to embed the JSON directly in the status, I think.

Reply:
I can put a JSON directly but what the advantage here to have it instead of a String? This field is something that the user has to read to understand if the proposal is fine with him. I don't expect any automation or application to intepret the JSON for making decision.

Reply:
Well JSON within a String within some JSON is just... icky. At least if it's an embedded JSON object the user can output it as YAML or pretty printed JSON so it's slightly more readable.